apps/experiments/models.py

```diff
@@ -1134,7 +1134,7 @@ def create_new_version(self, new_parent: Experiment) -> "ExperimentRoute":
                 # Compare experimens using their `version` instances for a comprehensive comparison
                 child_version = self.child.version
                 latest_version = latest_child_version.version
-                child_version.compare(latest_version)
+                child_version.compare(latest_version, early_abort=True)
```

Comment:
Won't this affect the `_generate_version_description` function?

Reply:
You are correct. Nice catch